etc/kayobe/seed-vm.yml

```diff
@@ -28,7 +28,6 @@ seed_vm_vcpus: 1
 # Base image for the seed VM root volume. Default is
 # "https://cloud.centos.org/centos/8/x86_64/images/CentOS-8-GenericCloud-8.2.2004-20200611.2.x86_64.qcow2".
 #seed_vm_root_image:
-seed_vm_root_image: https://cloud.centos.org/centos/8/x86_64/images/CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2
```

Comment:
I just got stuck on this issue - would be good to progress this change.

Reply:
I suppose this applies to Ussuri and Train also